### PR TITLE
Updated the environment variable for locale collation

### PR DESCRIPTION
**Pull request by jcoyne**
See https://www.gnu.org/software/libc/manual/html_node/Locale-Categories.html

### CODE REVIEW

File: specifications/cdx-format/openwayback-cdxj/index.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ is designed to facilitate this. A non-sorted CDXJ file is still valid as far as
 searching. 
 
 When sorted, the sorting should be done based on the native byte values of the characters. This is equivalent to using the GNU sort with 
-the collation settings `LC_ALL=C`.
+the collation settings `LC_COLLATION=C`.
```
Comment (ato) [inline specifications/cdx-format/openwayback-cdxj/index.md:202]: The libc doc linked in the description seems to mention LC_COLLATE not LC_COLLATION.

**Review by ato — CHANGES_REQUESTED** [review]

**Comment by wumpus** [comment]
This is a little confusing, I looked at my existing code and it uses LC_ALL=C. I have no idea when LC_COLLATE was added, but I don't think it existed decades ago. Perhaps safest to say mention both LC_COLLATE and LC_ALL.

**Comment by ato** [comment]
I think it'd be good to explain the motivation for this change in the commit message. LC_COLLATE and LC_ALL both ensure the sort order is correct. However, I'm assuming the motivation for recommending LC_COLLATE over LC_ALL is that LC_ALL will also change other unrelated things like the user-interface language. For users who use English as their UI language it makes little practical difference (which is why I lazily use LC_ALL even knowing the difference) but for those using a different UI language having sort suddenly output errors in English instead of their configured language may be undesirable.

**Comment by ato** [comment]
Actually I've just realised this is editing the openwayback cdxj specification document which is being removed entirely in #77.

File: specifications/cdx-format/openwayback-cdxj/index.md (hunk 1 above)
Comment (jcoyne) [reply specifications/cdx-format/openwayback-cdxj/index.md]: Thanks. I've updated that.

**Comment by ato** [comment]
The OpenWayback CDXJ proposal has been withdrawn. #77

**Pull request closed without merging**